source/_includes/common-tasks/backups.md

```diff
@@ -57,19 +67,60 @@ There are multiple ways to store the backup on another device:
-
-1. Select "Upload Backup" from the icon in the upper right of the page.
-2. Click on the folder icon to navigate to your backup .tar file and select it.
-
-When the upload is completed, you will be presented with the backup restore dialog for restoring it, and can then choose to restore in full or in part by manually selecting individual items.
-
-If the backup you are uploading is more than 1GB in size, it can be faster and more efficient to make use of the Samba add-on in order to transfer files to the `/backup` directory.
-
-The length of time it takes to create or restore backup will depend on how much you have to compress or decompress.
-
-If you're looking to slim down your backup, check if your configuration directory contains a large database file (`home-assistant_v2.db`). See the [`recorder`](/integrations/recorder/) integration page for options to keep your database data down to a size that won't cause issues. Note the keep days, purge interval, and include/exclude options.
-
-When the restore is complete, Home Assistant will restart to apply the new settings. You will lose the connection to the UI and it will return once the restart is completed.
+There are two ways to use a backup:
+
+- On your current system to recover your settings.
+- During onboarding, to migrate your setup to a new device or a to device on which you performed a factory reset and a new installation.
+
+#### Restoring a backup during onboarding
+
+You can use a backup during the onboarding process to restore your configuration.
+
+**Migration**: This procedure also works if you want to migrate from one device to another. In that case, use the backup of the old device on the new device. The target device can be a different device type. For example, you can migrate from a Raspberry&nbsp;Pi to another smart home hub.
+
+##### Prerequisites
+
+1. The login credentials of the device from which you made the backup.
+2. If you migrate the installation to a new device, make sure the new device has more storage capacity than the existing device.
+   - Before migrating, on the old system, check how much storage you used.
+     - Go to **{% my storage title="Settings > System > Storage" %}**, and under **Disk metrics**, hover over the **Used space** bar.
+     - The tooltip shows your disk size.
+     - The target device must have more free space than the source device.
+        - If your target device is a Home Assistant Yellow, note that it is the size of the eMMC that is relevant.
+        - The restore process mainly uses the eMMC, not the NVMe.
+3. If you are migrating to a new device:
+   - You do not need to transfer the backup to a USB or SD card to bring it to your smart home hub.
+   - You will be able to access the explorer of your PC.
+4. This procedure assumes you have already completed the [installation](/installation/) procedure on your target device and are now viewing the welcome screen as part of the [onboarding](/getting-started/onboarding/).
```

Comment:
Should this be a description right under the header instead of the last step?
You can not do anything in this list until you do this?

Reply:
You have a point there. On the other hand, the steps above are supposed to be done on the old system, before migrating. Also, this is a prerequisite section, for most of the steps, the order doesn't matter. Maybe I should make it a bulleted list.